docs/tools.md

```diff
@@ -186,6 +186,35 @@ Benchmark                               Time             CPU      Time Old
 This is a mix of the previous two modes, two (potentially different) benchmark binaries are run, and a different filter is applied to each one.
 As you can note, the values in `Time` and `CPU` columns are calculated as `(new - old) / |old|`.
 
+## Note: Interpreting the output
+
```

Comment:
This could use a snippet of `compare.py` output that contains said `pvalue` output.

Reply:
Eh, i think that could be useful, but it looks out of place there.
What i was thinking of was along the lines of `Example output` from above,
but one that does actually contain `pvalue`

Reply:
So, should I add an example like this? This is an hypothetical example.
```bash
Comparing BM_memcpy (from ./a.out) to BM_copy (from ./a.out)
Benchmark                               Time             CPU      Time Old      Time New       CPU Old       CPU New       P-value
--------------------------------------------------------------------------------------------------------------------------------------
[BM_memcpy vs. BM_copy]/8            +5.1649         +5.1637            37           230            37           230        0.001
[BM_memcpy vs. BM_copy]/64          +21.4352        +21.4374            74          1654            74          1653        0.0001
[BM_memcpy vs. BM_copy]/512        +143.6022       +143.5865            91         13122            91         13120        0.00001
```

Reply:
Let's add 
```
Benchmark                                                              Time             CPU      Time Old      Time New       CPU Old       CPU New
---------------------------------------------------------------------------------------------------------------------------------------------------
benchmark/threads:1/process_time/real_time_pvalue                    0.0000          0.0000      U Test, Repetitions: 27 vs 27
benchmark/threads:1/process_time/real_time_mean                     -0.1442         -0.1442            90            77            90            77
benchmark/threads:1/process_time/real_time_median                   -0.1444         -0.1444            90            77            90            77
benchmark/threads:1/process_time/real_time_stddev                   +0.3974         +0.3933             0             0             0             0
benchmark/threads:1/process_time/real_time_cv                       +0.6329         +0.6280             0             0             0             0
OVERALL_GEOMEAN                                                     -0.1442         -0.1442             0             0             0             0

```

Reply:
Hi Roman,

I added the provided example in addition, I also added an explanation summary of each output line.

https://github.com/google/benchmark/pull/1624/commits/0f64e48fcd8bace2a26d918d4b5c99ebb8258224